warehouse/templates/manage/token.html

```diff
@@ -70,15 +70,20 @@ <h2>Token for "{{ macaroon.description }}"</h2>
         <p id="description-help-text" class="form-group__help-text">What is this token for?</p>
       </div>
       <div class="form-group">
-        <label for="scope" class="form-group__label">Scope</label>
-        <select name="token_scope" id="scope" class="form-group__input">
+        <label for="token_scope" class="form-group__label">Scope</label>
+        <select name="token_scope" id="token_scope" class="form-group__input" aria-describedby="token_scope-errors">
+          <option hidden selected value="scope:unspecified">Select scope...</option>
```

Comment:
```suggestion
          <option disabled selected value="scope:unspecified">Select scope...</option>
```

I think it's better here if this is visible, but disabled. As is, when the user opens the select box, "Entire account" already appears to be selected, but it's not _actually_ selected until the user clicks on the option.

Before:
<img width="383" alt="Screen Shot 2019-07-29 at 10 43 40 AM" src="https://user-images.githubusercontent.com/294415/62057499-c27f5100-b1ed-11e9-8b22-9db8ef251e4e.png">

After:
<img width="369" alt="Screen Shot 2019-07-29 at 10 43 24 AM" src="https://user-images.githubusercontent.com/294415/62057509-cad78c00-b1ed-11e9-9462-dadc6f2e124e.png">